APP/lib/api_service.dart

```diff
@@ -816,14 +816,12 @@ class ApiService {
     required String group,
     required String event,
   }) async {
+    final cacheKey = 'scouting_report';
```

Comment:
```suggestion
    final cacheKey = '${group}_${event}_scouting_report';
```
Make sure that you cache the different scouting reports under different keys